packages/core/src/RenderingEngine/StackViewport.ts

```diff
@@ -1755,27 +1778,30 @@ class StackViewport extends Viewport implements IStackViewport {
     }
 
     this.initialVOIRange = voiRange;
+    this.voiFunction = VOILUTFunctionType[voiLUTFunction];
```

Comment:
If `imagePixelModule` includes the `voiLUTFunction` as the Enum, we don't need to do this here and can just simply assign